gtsfm/frontend/correspondence_generator/keypoint_aggregator/keypoint_aggregator_dedup.py

```diff
@@ -15,11 +15,17 @@
 
 
 class KeypointAggregatorDedup(KeypointAggregatorBase):
-    """Keypoint aggregator with de-duplication."""
+    """Keypoint aggregator with de-duplication of keypoints within each image."""
 
-    def __init__(self) -> None:
-        """Initialize global variables"""
+    def __init__(self, nms_merge_radius: float = 3) -> None:
+        """Initialize global variables.
+
+        Args:
+            nms_merge_radius: Radius (in pixels) to use when merging detections within the same view (image).
+                Note that tracks are merged, not suppressed.
+        """
         self.duplicates_found = 0
+        self.nms_merge_radius = nms_merge_radius
 
     def append_unique_keypoints(
         self, i: int, keypoints: Keypoints, per_image_kpt_coordinates: Dict[Tuple[int, int], np.ndarray]
```

Comment:
I believe the type hint for `per_image_kpt_coordinates` should be `Dict[int, np.ndarray]`

Reply:
Fixed, thanks.